src/workloads/sharding/WriteOneWithoutShardKeyUnshardedCollection.yml

```diff
@@ -0,0 +1,105 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  On an unsharded collection on a single shard cluster, run workloads that updateOne, deleteOne, and findAndModify.
+
+GlobalDefaults:
+  Nop: &Nop {Nop: true}
+
+  Database: &Database test
+  UnshardedCollection: &UnshardedCollection Collection1
```

Comment:
Since this is an unsharded collection test with only unsharded collections, we can probably remove the Unsharded prefix and just keep it as collection so as to not be so verbose. Same as above.